packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -263,20 +254,6 @@ const IDVForm = ({
                                         </fieldset>
                                     )}
                                 </div>
-                                {document_image && (
-                                    <div className='proof-of-identity__sample-container'>
-                                        <Text size='xxs' weight='bold'>
-                                            {localize('Sample:')}
-                                        </Text>
-                                        <div className='proof-of-identity__image-container'>
```

Comment:
should we remove the corresponding styles if they're not being used anywhere else?

Reply:
Let me look at this